legos/ctftime.py

```diff
@@ -59,8 +66,7 @@ def _handle_opts(message):
 
     def _get_upcoming(self, message):
 
-        baseurl = 'https://ctftime.org/api/v1'
-        r = requests.get('{}/events/'.format(baseurl))
+        r = requests.get('{}/events/'.format(API_URL))
```

Comment:
Thanks for making this a global. I was trying to make it a class variable but kept getting tripped up on the init super() stuff.